.gitignore

```diff
@@ -16,4 +16,11 @@
 *.out
 *.app
 example
-./shell
+/vic
```

Comment:
@mrshu 'A leading slash matches the beginning of the pathname.' It works like a charm